Week01/Problem02/awillats_02.py

```diff
@@ -0,0 +1,33 @@
+#By considering the terms in the Fibonacci sequence whose values do not exceed four million,
+# find the sum of the even-valued terms.
+
+import math
+#max_num = int(1000)
+max_num = int(4e6)
+
+old_fib = 0
+fib = 1
+new_fib = 0
+
+sum_fib = 0
+
+#while fib < max_num:
+
+#needed to check before sum
+#then fibs were wrong
+
+#printed to check fibs, figured out
+
+while fib <= max_num:
+
+    print(fib,"+",old_fib,"=",fib+old_fib)
```

Comment:
I think you could use string formatting to make your life easier here.  Python supports two "flavors" of string formatting, both [C-style](https://www.learnpython.org/en/String_Formatting) and specifically [Python 3 style](http://www.python-course.eu/python3_formatted_output.php); both styles are compared [here](https://pyformat.info/).